plugins/bios/fu-plugin-bios.c

```diff
@@ -53,10 +53,30 @@ fu_plugin_bios_coldplug(FuPlugin *plugin, GError **error)
 	return TRUE;
 }
 
+static void
+fu_plugin_bios_add_security_attrs(FuPlugin *plugin, FuSecurityAttrs *attrs)
+{
+#ifdef HAVE_HSI
```

Comment:
LGTM, thanks. Out of interest, why the #ifdef? If there's no HSI then the vfunc just won't ever be called -- it might be best to just leave it compiled in.

Reply:
Good point, I'll drop it.